packages/input_schema/src/utilities.ts

```diff
@@ -137,12 +137,12 @@ export function validateInputUsingValidator(
     }
 
     Object.keys(properties).forEach((property) => {
-        const value = input[property] as Record<string, any>;
-        const { type, editor, patternKey, patternValue } = properties[property];
+        const value = input[property] as Record<string, any> | Array<any> | string | number | boolean | null;
```

Comment:
I wonder if you really need this long type hint, maybe `unknown` would be just fine, since you have all the type guards anyway.